cmd/drone-server/server.go

```diff
@@ -513,6 +521,16 @@ var flags = []cli.Flag{
 		Name:   "keepalive-min-time",
 		Usage:  "server-side enforcement policy on the minimum amount of time a client should wait before sending a keepalive ping.",
 	},
+	cli.BoolFlag{
+		EnvVar: "WOODPECKER_GITEA_REV_PROXY_AUTH",
+		Name:   "gitea-rev-proxy-auth",
+		Usage:  "enable gitea authentication using HTTP header specified in WOODPECKER_GITEA_REV_PROXY_AUTH_HEADER",
+	},
+	cli.StringFlag{
+		EnvVar: "WOODPECKER_GITEA_REV_PROXY_AUTH_HEADER",
+		Name:   "gitea-rev-proxy-auth-header",
```

Comment:
wonder if we can make it more generic?

Reply:
Please propose better name (WOODPECKER_TO_VCS_AUTH_HEADER maybe)?

Reply:
`WOODPECKER_PROXY_AUTH_HEADER` `proxy-auth-header` ?

Reply:
> WOODPECKER_PROXY_AUTH_HEADER proxy-auth-header ?

Ambiguous - one may think its header from rev proxy to woodpecker (its header sent from woodpecker to vcs).

Reply:
Hmm let me rethink